component-samples/scripts/onDieCache.py

```diff
@@ -0,0 +1,82 @@
+#This script copies OnDie ECDSA artifacts from the cloud to a specified local directory.
+#Typically, the FDO components (Mfg, RV, Owner) will be configured to load OnDie certs from this
+#local directory. 
+#
+#When files are updated in the local directory, they will have a file type of ".new". In addition,
+#a touch file is created after copying to indicate to the component than an update has been made.
+#The component will trigger an update when this touch file is detected and will rename the .new 
+#files to the original names as the cache files are read. 
+#
+#The use of the .new mechanism is employed to prevent any conflict between reading a file and 
+#copying a new version from the script.
+
+import sys, getopt
+import urllib.request
+import argparse
+import os.path
+import zipfile
+import glob
+import shutil
+
+def artifact_copy(source_pathname, destdir):
```

Comment:
This script works with python 3 (python 3.8.2 to be specific) but not with python 2 ( tested with python 2.7). It would help if we can include that info in the script or somewhere in the README

Reply:
Updated comments in script.

Reply:
Could you please help me with the line number, I am unable to find it.

Reply:
Latest commit should have the change now.

Reply:
I see now.